testsuite/tests/kuadrant/authorino/operator/http/test_raw_http.py

```diff
@@ -0,0 +1,18 @@
+"""
+Test raw http authorization interface.
+"""
+
+import pytest
+
+def test_authorized_via_http(authorization, client_http_auth, auth):
+    response = client_http_auth.request("GET", "/check", auth=auth, json='{}')
+    assert response.is_success
+    assert response.status_code == 200
+    assert response.text == ''
+    assert response.headers.get('x-ext-auth-other-json', '') == '{"propX":"valueX"}'
+
+def test_unauthorized_via_http(authorization, client_http_auth):
+    response = client_http_auth.request("GET", "/check", json='{}')
+    assert not response.is_success
+    assert response.status_code == 401
+    assert response.text == ''
```

Comment:
The text is important in this one, right?

Reply:
What do you mean exactly? I think it is needed to check as much as we can. On the other hand Authorino returns some explaining text in case of not successful call. Do you think that text is missing in this case?

Reply:
Sorry, the question should be `The text is NOT important in this one, right?`. Authorino uses Header for extra information about checks not response text I believe, so this check is not needed.